test/Tester/PSClientTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Orleans;
+using Orleans.Runtime;
+using OrleansPSUtils;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+using UnitTests.Tester;
+using Xunit;
+
+namespace Tester
+{
+    public class PowershellHostFixture : DefaultClusterFixture
+    {
+        public PowerShell Powershell { get; set; }
+        public Runspace Runspace { get; set; }
+
+        public PowershellHostFixture()
+        {
+            var initialSessionState = InitialSessionState.CreateDefault();
+            initialSessionState.Commands.Add(new SessionStateCmdletEntry("Start-GrainClient", typeof(StartGrainClient), null));
+            initialSessionState.Commands.Add(new SessionStateCmdletEntry("Stop-GrainClient", typeof(StopGrainClient), null));
+            initialSessionState.Commands.Add(new SessionStateCmdletEntry("Get-Grain", typeof(GetGrain), null));
+            Runspace = RunspaceFactory.CreateRunspace(initialSessionState);
+            Runspace.Open();
+            Powershell = PowerShell.Create();
+            Powershell.Runspace = Runspace;
+        }
+
+        public override void Dispose()
+        {
+            var stopCommand = new Command("Stop-GrainClient");
+            Powershell.Commands.Clear();
+            Powershell.Commands.AddCommand(stopCommand);
+            Powershell.Invoke();
+            Powershell.Dispose();
+            Runspace.Dispose();
+            base.Dispose();
+        }
+    }
+
+    public class PSClientTests : OrleansTestingBase, IClassFixture<PowershellHostFixture>
+    {
+        private PowerShell _ps;
+
+        public PSClientTests(PowershellHostFixture fixture)
+        {
+            this._ps = fixture.Powershell;
+            _ps.Commands.Clear();
+        }
+
+        [Fact, TestCategory("BVT"), TestCategory("Tooling")]
+        public void GetGrain()
+        {
+            var getGrainCommand = new Command("Get-Grain");
+            getGrainCommand.Parameters.Add("GrainType", typeof(IManagementGrain));
+            getGrainCommand.Parameters.Add("LongKey", RuntimeInterfaceConstants.SYSTEM_MANAGEMENT_ID);
+            _ps.Commands.AddCommand(getGrainCommand);
+
+            var results = _ps.Invoke<IManagementGrain>();
```

Comment:
Yes, this is the case. Since many other tests cover the GrainClient itself, I just made a simple initial test to make sure the powershell is executing the correct things. Can add test for the others and use an exclusive silo for that.